src/lib.rs

```diff
@@ -91,6 +91,18 @@ where
     }
 }
 
+pub fn is_lambda() -> bool {
+    #[cfg(feature = "local")]
+    {
+        return std::env::var_os("AWS_LAMBDA_RUNTIME_API").is_some();
```

Comment:
These shouldn't be returns:
```suggestion
        std::env::var_os("AWS_LAMBDA_RUNTIME_API").is_some()
```